guides/plugins/plugins/content/media/remote-thumbnail-generation.md

```diff
@@ -0,0 +1,50 @@
+---
+nav:
+  title: Remote Thumbnail Generation
+  position: 40
+
+---
+
+# Remote Thumbnail Generation
+
+In certain scenarios, you might want to disable the filesystem thumbnail generation in Shopware and use an external CDN service to handle the thumbnails.
+This can be beneficial for performance and scalability reasons.
+
+Starting with version 6.6.4.0, Shopware introduces the feature of remote thumbnail generation.
+When the remote thumbnail configuration is enabled, the thumbnail images and thumbnail records in the database are not generated by Shopware, but by the external service.
+
+## Configuration
+
+To use remote thumbnails, you need to adjust the following parameters in your `config/packages/shopware.yaml`:
+
+```yaml
+media:
```

Comment:
@tamvt can you add the shopware key here as well. Many people just copy the code.